doc/engineering-reference/src/special-modules-reporting/resilience-metrics.tex

```diff
@@ -66,69 +66,17 @@ \subsubsection{Heat Index}\label{heat-index}
 \end{tabular}
 \end{table}
 
-The computation of the heat index is a refinement of a result obtained by
+Before version 24.2, the computation of the heat index is a refinement of a result obtained by
 multiple regression analysis carried out by Lans P. Rothfusz and described in a
-1990 National Weather Service (NWS) Technical Attachment (SR 90-23) [4-5]. The
-calculation is based on degree Fahrenheit.
+1990 National Weather Service (NWS) Technical Attachment (SR 90-23) [4-5].
 
-The regression equation of Rothfusz is
-\begin{equation}  \label{eq:rm-1}
-HI = c_1 + c_2T + c_3R + c_4TR + c_5T^2 + c_6R^2 + c_7T^2R + c_8TR^2 + c_9T^2R^2
-\end{equation}
-
-where
-
-HI = heat index (expressed as an apparent temperature in degrees Fahrenheit),
-
-T = ambient dry-bulb temperature (in degrees Fahrenheit),
-
-R = relative humidity (percentage value between 0 and 100),
-
-$c_1$ = -42.379,
-
-$c_2$ = 2.04901523,
-
-$c_3$ = 10.14333127,
-
-$c_4$ = -0.22475541,
-
-$c_5$ = -0.00683783,
-
-$c_6$ = -0.05481717,
-
-$c_7$ = 0.00122874,
-
-$c_8$ = 0.00085282,
-
-$c_9$ = -0.00000199.
-
-If the RH is less than 13\% and the temperature is between 80 and \IP{112}{\fahrenheit}, then
-the following adjustment is subtracted from HI:
-
-\begin{equation}  \label{eq:rm-2}
-HI = (13 - R) / 4 * ((17 - |T - 95|) / 17)^{0.5}
-\end{equation}
-
-Otherwise, if the RH is greater than 85\% and the temperature is between 80 and
-\IP{87}{\fahrenheit}, then the following adjustment is added to HI:
-
-\begin{equation}  \label{eq:rm-3}
-HI = (R - 85) / 10 * (87 - T) / 5
-\end{equation}
-
-The Rothfusz regression is not appropriate when conditions of temperature and
-humidity warrant a heat index value below about \IP{80}{\fahrenheit}. In those cases, a simpler
-formula is applied to calculate values consistent with Steadman's results:
-
-\begin{equation}  \label{eq:rm-4}
-HI = 0.5 * (T + 61.0 + (T - 68.0) * 1.2 + (R * 0.094))
-\end{equation}
-
-In practice, the simple formula is computed first based on the temperature and
-humidity. If this heat index value is \IP{80}{\fahrenheit} or higher, the full regression
-equation along with any adjustment as described above is applied. The Rothfusz
-regression is not valid for extreme temperature and relative humidity conditions
-beyond the range of data considered by Steadman.
+Starting from version 24.2, the heat index calculation adopts the extended heat
+index method developed by Lu & Romps [17]. The previous heat index gives
```

Comment:
There is a LaTeX error here with the & character, it needs to be escaped.

Reply:
Thanks. I just escaped it.